### PR TITLE
Add mutable support

### PR DESCRIPTION
**Pull request by 10maurycy10**
add make_static_mut()

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -9,3 +12,10 @@ pub fn make_static<'a, T>(input: &'a T) -> &'static T {
     let f: fn(_, &'a T) -> &'static T = helper;
     f(&&(), input)
 }
+
+/// Use black magic fuckery to change the lifetime on a mutable refrence
+/// May introduce undefined behavior.
+pub fn make_static_mut<'a, 'b, T>(input: &'a mut T) -> &'b static T {
+    let f: fn(_, &'a mut T) -> &'b static T = helper_mut;
```
Comment (andersk) [inline src/lib.rs]: This isn’t valid syntax; you mean `&'b mut T`.

```suggestion
pub fn make_static_mut<'a, 'b, T>(input: &'a mut T) -> &'b mut T {
    let f: fn(_, &'a mut T) -> &'b mut T = helper_mut;
```

Comment (10maurycy10) [reply src/lib.rs]: Fixed.

**Comment by TmLev** [comment]
Guys, can we merge this? Really need this in prod, like, yesterday

**Comment by meithecatte** [comment]
> Guys, can we merge this? Really need this in prod, like, yesterday

Please contact `sales@compilercrim.es` if you'd be interested in a support contract for this issue :+1:

**Pull request closed without merging**